### PR TITLE
lucetc: report all unresolved bindings, not just the first one

### PR DESCRIPTION
**Pull request by jedisct1**

### CODE REVIEW

**Comment by fst-crenshaw** [comment]
Thanks for this quick work to improve the error output that lucet provides!  It will help users to know the full list of unresolved bindings.  

One of the things that bit me a little bit over in https://github.com/bytecodealliance/lucet/pull/374 was the lack of tests on error output.  I ended up squashing too much error output.  😿 

With that in mind, I wonder if you might have plans to add a test for this work?

**Comment by jedisct1** [comment]
Is `anyhow` guaranteed to always print errors the same way, byte for byte?

File: lucet-module/src/error.rs [hunk 1]
```diff
@@ -31,4 +32,6 @@ pub enum Error {
     UnknownModule { module: String, symbol: String },
     #[error("Unknown symbol `{module}::{symbol}`")]
     UnknownSymbol { module: String, symbol: String },
+    #[error("Unknown bindings {unknown_bindings}")]
+    UnknownBindings { unknown_bindings: UnknownBindings },
```
Comment (acfoltzer) [inline lucet-module/src/error.rs]: If you add a `#[from]` here, decls.rs:201 could be a bit simpler

Comment (jedisct1) [reply lucet-module/src/error.rs]: Done, thanks!

**Comment by pchickey** [comment]
@jedisct1 for testing I'd reuse the existing `import_many.wat` test and match on the `lucetc::Error`, rather than coerce it to an anyhow::Error.

**Comment by pchickey** [comment]
This PR was closed as a byproduct of deleting the branch named `master`. If this is still an active PR, re-open as a new PR against `main`.

**Pull request closed without merging**